benchmarks/continuous.py

```diff
@@ -89,6 +165,14 @@ class Result:
   measure: str
   value: Union[int, float]
 
+  def compare(self, other):
+    if self.measure == 'alloc':
+      return self
+    if self.measure == 'time':
+      return Result(self.benchmark, 'time_rel', self.value / other.value)
+    if self.measure == 'runtime':
+      return Result(self.benchmark, 'runtime_rel', self.value / other.value)
```

Comment:
raise an error if none of those ifs take?

Reply:
Done.